game/test/contrib/utils/components/skill/SkillTest.java

```diff
@@ -0,0 +1,62 @@
+package contrib.utils.components.skill;
+
+import static org.junit.Assert.*;
+
+import core.Entity;
+import core.Game;
+
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+public class SkillTest {
+
+    private static int value = 0;
+
+    private Consumer<Entity> skillFunction = entity -> value++;
+
+    @After
+    public void cleanup() {
+        value = 0;
+        Game.removeAllEntities();
+    }
+
+    @Test
+    public void execute() {
+        Entity entity = new Entity();
+        final int baseCoolDownInSeconds = 2;
+        Skill skill = new Skill(skillFunction, baseCoolDownInSeconds);
+
+        assertTrue("Skill should be executable", skill.canBeUsedAgain());
+        skill.execute(entity);
+        assertEquals("Skill should have been executed once", 1, value);
+        assertFalse("Skill is in cool down and can not be executable", skill.canBeUsedAgain());
+    }
+
+    @Test
+    public void executeWhenCoolDownActive() {
+        Entity entity = new Entity();
+        final int baseCoolDownInSeconds = 2;
+        Skill skill = new Skill(skillFunction, baseCoolDownInSeconds);
+
+        skill.execute(entity);
+        assertFalse("Skill should not be executable", skill.canBeUsedAgain());
+        assertEquals("Skill should have been executed once", 1, value);
```

Comment:
```suggestion
        skill.execute(entity);
        assertEquals("Skill should have been executed once", 1, value);
```
Oder irre ich mich da sonst ja kein Ausführen stattfindet während der Cooldown.